docs/howto/windows.md

```diff
@@ -106,28 +87,132 @@ many of these.)
   but it may be a bit simpler to use the Linux version.
 * One significant limitation of WSL (as of 2018) is that doing a large
   amount of filesystem I/O tends to be slow.  This is a reason to do
-  operations like a Gradle build, perhaps `git` commands, and perhaps
+  operations like a Gradle build ( Gradle is the Java program that does the Android build), perhaps `git` commands, and perhaps
   the occasional `yarn install` using the Windows versions.
 
 [so-guide]: https://stackoverflow.com/questions/42614347/running-react-native-in-wsl-with-the-emulator-running-directly-in-windows
 
-### Accessing the filesystem
-You can access your Windows local drives from `bash` and other Linux
-programs by navigating to `/mnt/<drive-letter>` .
 
-### Running `yarn` commands
-First, install Node.js and Yarn in your Linux environment.  You'll follow
-the same steps as in [our main setup guide](build-run.md) -- just use the
-Linux versions. :-)
+### Step-by-step instructions for setting up WSL
+
+1. Open Powershell as an Administrator and run the command below to enable WSL. 
+
+    `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux`
+
+2. Install [Chocolatey][chocolatey]. After installing Chocolatey, use it to install the JAVA SE Development Kit with  
```

Comment:
This is one I'm particularly curious about, because I have actually never heard of Chocolatey!

From the web, I see it's a popular package manager for Windows. Did you mention Chocolatey here because it's just what you use routinely to install software? Or is there something more specific about why Chocolatey in particular?

Reply:
I had never heard of Chocolatey before this project either. It was actually recommended by the [React Native](https://facebook.github.io/react-native/docs/getting-started.html#node-python2-jdk) Official Docs.